run.sh

```diff
@@ -6,7 +6,7 @@ is_installed() {
 }
 
 # Install pigpio packages if not already installed
-for pkg in libpigpio-dev libpigpiod-if-dev pigpio; do
+for pkg in libpigpiod-if2-1 pigpio; do
```

Comment:
I was previously trying to say all this could be replaced by one single apt command...

Also, I think pigpio just pulls in all of this. So `apt-get install -qqy pigpio` would be all that's needed. No for loop, no "is_installed" check, etc.